code/src/App.js

```diff
@@ -1,12 +1,27 @@
 import React from 'react'
 import data from './data.json'
+import {Album} from './Album.js'
+import {Header} from './Header'
+import './index.css'
 
 console.log(data)
```

Comment:
remember to remove console.logs  from your code after you finish debugging your code.